app/src/lib/stores/app-store.ts

```diff
@@ -4711,6 +4741,21 @@ export class AppStore extends TypedBaseStore<IAppState> {
     await createDesktopStashEntry(repository, branchName, tip.branch.tip.sha)
   }
 
+  public async _popStash(repository: Repository, branch: Branch) {
+    const stash = await getLastDesktopStashEntryForBranch(
+      repository,
+      branch.name
+    )
+
+    if (stash === null) {
+      log.warn('no stash found that matches')
+      return
+    }
+
+    await popStashEntry(repository, stash.stashSha)
+    log.warn(`Popped stash with commit id ${stash.stashSha}`)
```

Comment:
this is still `warn`?
```suggestion
    log.info(`Popped stash with commit id ${stash.stashSha}`)
```